poker-base.cabal

```diff
@@ -27,6 +27,7 @@ source-repository head
 library
   exposed-modules:
     Poker
+    Poker.Evaluate
```

Comment:
How about HandRank? I think Evaluate is quite an overloaded word (like monte-carlo equity for example)